cmake/PACE_Docs.cmake

```diff
@@ -1,55 +1,120 @@
-# Build documentation
+#[=======================================================================[.rst:
+PACE_Docs
+-----------------
 
-set(DOCS_ROOT_DIR "${Horace_ROOT}/documentation/user_docs")
-set(DOCS_SOURCE_DIR "${DOCS_ROOT_DIR}/docs")
-set(DOCS_WORK_DIR "${DOCS_ROOT_DIR}/build" CACHE FILEPATH "Directory to put in-progress docs")
-set(DOCS_OUTPUT_DIR "${DOCS_WORK_DIR}/html" CACHE FILEPATH "Directory containing built HTML documentation")
-set(MANUAL_WORK_DIR "${DOCS_WORK_DIR}/latex" CACHE FILEPATH "Directory to build LaTeX sources")
-set(MANUAL_OUTPUT_DIR "${DOCS_WORK_DIR}/latex" CACHE FILEPATH "Directory to put compiled LaTeX manual")
+Build Horace user documentation as either HTML pages or a LaTeX manual
+
+Variables required by the module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``${Horace_ROOT}``
+This is provided by the main CMakeLists.txt
+
+Variables defined by the module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``Horace_DOCS_WORK_DIR``
+Working directory where HTML docs are to be built
+
+``Horace_DOCS_OUTPUT_DIR``
+Output directory where HTML docs are placed
+
+``Horace_DOCS_PACK_OUTPUT``
+Output filename (including extension) for compressed docs file
+e.g. ``${CMAKE_CURRENT_BINARY_DIR}/docs.zip``
+
+``Horace_MANUAL_WORK_DIR``
+Working directory where LaTeX docs are to be built
+
+``Horace_MANUAL_OUTPUT_DIR``
+Output directory where LaTeX docs are placed
```

Comment:
is this an output or working directory? or is this where the built `.pdf` or `.dvi` is placed?